packages/datetime/src/dateRangeInput.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+import * as classNames from "classnames";
+import * as React from "react";
+
+import {
+    AbstractComponent,
+    Button,
+    Classes,
+    InputGroup,
+    Intent,
+    IProps,
+} from "@blueprintjs/core";
+
+import * as DateClasses from "./common/classes";
+import {
+    IDatePickerBaseProps,
+} from "./datePickerCore";
+
+export interface IDateRangeInputProps extends IDatePickerBaseProps, IProps {}
+
+export class DateRangeInput extends AbstractComponent<IDateRangeInputProps, {}> {
+    public static defaultProps: IDateRangeInputProps = {};
+
+    public displayName = "Blueprint.DateRangeInput";
+
+    private startDateInputRef: HTMLInputElement = null;
+    private endDateInputRef: HTMLInputElement = null;
+
+    public render() {
+        return (
+            <div className={Classes.CONTROL_GROUP}>
+                <InputGroup
+                    className={DateClasses.DATERANGEINPUT_FIELD}
+                    inputRef={this.setStartDateInputRef}
+                    placeholder="Start date"
+                    type="text"
+                />
+                <InputGroup
+                    className={DateClasses.DATERANGEINPUT_FIELD}
+                    inputRef={this.setEndDateInputRef}
+                    placeholder="End date"
+                    type="text"
```

Comment:
how about `leftInputProps` and `rightInputProps` for passing arbitrary props to these two fields? that'll cover like 1000 bases ⚾️.